liblangutil/Scanner.cpp

```diff
@@ -429,6 +429,7 @@ Token Scanner::scanSlash()
 			Token comment;
 			m_skippedComments[NextNext].location.start = firstSlashPosition;
 			comment = scanSingleLineDocComment();
+			m_skippedComments[NextNext].location.source = m_source;
```

Comment:
The node factory requires a valid source for a location when creating a new node. Also needed for proper error reporting later on.

Reply:
Why not assign it rigth after start (all others need `scanSingleLineDocComment` to run first)? The same below.

Reply:
Done.